bndtools.core/src/bndtools/launch/util/LaunchUtils.java

```diff
@@ -20,13 +20,8 @@ private LaunchUtils() {}
 
     public static IResource getTargetResource(ILaunchConfiguration configuration) throws CoreException {
         String target = getTargetName(configuration);
-        if (target == null)
```

Comment:
This null check is still necessary since you do not want to pass null to findMember.